src/app/space/settings/collaborators/collaborators.component.ts

```diff
@@ -23,8 +24,9 @@ export class CollaboratorsComponent implements OnInit, OnDestroy {
   private contextSubscription: Subscription;
   private collaboratorSubscription: Subscription;
   private userToRemove: User;
-  @ViewChild('addCollaborators') addCollaboratorsModal: IModalHost;
-  @ViewChild('removeCollaborator') removeCollaborator: IModalHost;
+  @ViewChild('addCollabDialog') addCollabDialog: AddCollaboratorsDialogComponent;
+  @ViewChild(ModalDirective) modalAdd: ModalDirective;
+  @ViewChild(ModalDirective) modalDelete: ModalDirective;
```

Comment:
I think the `@ViewChild(ModalDirective)` here is the culprit for the modal mixup above.